cloudflare/resource_cloudflare_load_balancer.go

```diff
@@ -434,6 +611,158 @@ func resourceCloudflareLoadBalancerImport(d *schema.ResourceData, meta interface
 	return []*schema.ResourceData{d}, nil
 }
 
+func flattenRules(rules []*cloudflare.LoadBalancerRule) (interface{}, error) {
+
+	if len(rules) == 0 {
+		return nil, nil
+	}
+
+	bb, err := json.Marshal(rules)
+	if err != nil {
+		return nil, err
+	}
+
+	var rout []interface{}
+	if err := json.Unmarshal(bb, &rout); err != nil {
+		return nil, err
+	}
```

Comment:
i don't quite understand why we are marshaling and unmarshaling here; can't we just iterate over the rules and add them to the return value? similar to something like https://github.com/cloudflare/terraform-provider-cloudflare/blob/master/cloudflare/resource_cloudflare_api_token.go#L179 or another existing resource?